### PR TITLE
feat: init input component

### PR DESCRIPTION
**Pull request by iDestin**
init input component #7

### CODE REVIEW

**Comment by FliPPeDround** [comment]
I think we should add a corresponding control to each Variant

**Comment by iDestin** [comment]
ok! I will try to complete

**Comment by FliPPeDround** [comment]
> ok! I will try to complete

Best to develop after updating the branch

File: src/components/Form/input.story.vue [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+const input4 = ref('')
+const textarea = ref('')
+const textarea1 = ref('')
+const textarea2 = ref('')
+const select = ref('')
+const text = ref('')
+</script>
+
+<template>
+  <Story title="Form/Input">
```
Comment (FliPPeDround) [inline src/components/Form/input.story.vue]: with type: 'grid' you can specify suitable width
eg:
<Story title="From/Input" layout="{type: "grid", width: 500}">

**Comment by iDestin** [comment]
please cc ? @FliPPeDround

**Pull request merged**